CMakeLists.txt

```diff
@@ -256,6 +259,11 @@ set(viam_rust_utils_file ${CMAKE_CURRENT_BINARY_DIR}/${CMAKE_SHARED_LIBRARY_PREF
 
 file(GLOB viam_rust_utils_files ${PROJECT_SOURCE_DIR}/${CMAKE_SHARED_LIBRARY_PREFIX}viam_rust_utils*${CMAKE_STATIC_LIBRARY_SUFFIX})
 
+# Check if the architecture is one that we support in rust-utils
+set(WINDOWS_ARCHS "AMD64" "X86" "X64")
```

Comment:
Per this SO answer: https://stackoverflow.com/a/70498851, there are only four values possible on windows: `AMD64`, `IA64`, `ARM64`, `EM64T`, and `X86`.

- You can safely ignore `IA64` (Itanium) and `EM64T` (not seen since XP).
- That leaves `AMD64` and `ARM64` for 64-bit Intel and ARM, and `x86` for 32-bit Intel Windows.

I doubt rust-utils is going to support 32-bit Windows or 64-bit ARM windows?

So it feels to me like the only value that matters here is `AMD64`, in which case you can just test that directly down on 277.